config/foreman.migrations/20200113133837_reset_bind-host-foreman-proxy.rb

```diff
@@ -0,0 +1,6 @@
+if answers['foreman_proxy']['bind_host'].is_a?(Array) &&
```

Comment:
`answers['foreman_proxy']` can be `false` in which case it crashes. I don't think we can use `answers.dig('foreman_proxy', 'bind_host')` in Ruby 2.0 yet, but that would be the defensive way.

Reply:
Added an `is_a?(Hash)` check